example/lib/local_style.dart

```diff
@@ -55,21 +55,26 @@ class LocalStyleState extends State<LocalStyle> {
 
   @override
   Widget build(BuildContext context) {
-    final styleAbsoluteFilePath = this.styleAbsoluteFilePath;
-
     if (styleAbsoluteFilePath == null) {
       return const Scaffold(
         body: Center(child: Text('Creating local style file...')),
       );
     }
 
     return Scaffold(
+        floatingActionButton: FloatingActionButton.small(
+          child: const Icon(Icons.layers_outlined),
+          onPressed: () async {
+            if (styleAbsoluteFilePath != null) {
+              await mapController?.setStyle(styleAbsoluteFilePath!);
+            }
+          },
+        ),
         body: MapLibreMap(
-      styleString: styleAbsoluteFilePath,
-      onMapCreated: _onMapCreated,
-      initialCameraPosition: const CameraPosition(target: LatLng(0.0, 0.0)),
-      onStyleLoadedCallback: onStyleLoadedCallback,
-    ));
+          onMapCreated: _onMapCreated,
+          initialCameraPosition: const CameraPosition(target: LatLng(0.0, 0.0)),
+          onStyleLoadedCallback: onStyleLoadedCallback,
+        ));
```

Comment:
Adding the option to change the map to this example could be confusing because this example showcases that the map style can in general be loaded from an local source. Could you create a new page that showcases this functionality please?